src/webgl/context-state.js

```diff
@@ -138,6 +138,18 @@ export function setParameters(gl, parameters) {
 export function withParameters(gl, parameters, func) {
   // assertWebGLContext(gl);
 
+  let emptyParametes = true;
+  /* eslint-disable no-unused-vars  */
+  for (const key in parameters) {
+    emptyParametes = false;
+    break;
+  }
+  /* eslint-enable no-unused-vars  */
+
+  if (emptyParametes) {
```

Comment:
Comment: `// Avoid setting state if no parameters provided. Just call and return`